packages/hoprd/src/index.ts

```diff
@@ -194,10 +195,14 @@ async function generateNodeOptions(environment: ResolvedEnvironment): Promise<Ho
 }
 
 function addUnhandledPromiseRejectionHandler() {
-  process.on('unhandledRejection', (reason: any, promise: Promise<any>) => {
-    console.error('Unhandled Rejection at:', promise, 'reason:', reason)
-    process.exit(1)
-  })
+  if (NODE_ENV === 'development') {
+    require('trace-unhandled/register')
```

Comment:
Why should we not use this in production too? Does it expose any sensible data? In essence its just a better log line.

Reply:
Would be great to have this in prod too